WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -0,0 +1,57 @@
+
+## Daily routine
+
+1. Start
+2. Check code review requests. [link](https://github.com/zkSNACKs/WalletWasabi/pulls) __Reviews / Awaiting review from you__
+2. Check [priority issues!](https://github.com/zkSNACKs/WalletWasabi/pulls?q=is%3Aopen+is%3Apr+label%3Apriority). Take it or set Assignees and Labels if you can.
```

Comment:
Incorrect link.
```suggestion
2. Check [priority issues!](https://github.com/issues?q=is%3Aopen+archived%3Afalse+user%3AzkSNACKs+label%3Apriority). Take it or set Assignees and Labels if you can.
```

Reply:
The proposed link contains PRs also. We can use this one instead.
https://github.com/zkSNACKs/WalletWasabi/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3Apriority+

Reply:
Fixed.

Reply:
This is not fixed.